pkg/clustermanager/eksa_installer.go

```diff
@@ -38,7 +38,7 @@ func NewEKSAInstaller(client KubernetesClient, reader manifests.FileReader) *EKS
 }
 
 // Install configures and applies eks-a components in a cluster accordingly to a spec.
-func (i *EKSAInstaller) Install(ctx context.Context, log logr.Logger, cluster *types.Cluster, spec *cluster.Spec) error {
+func (i *EKSAInstaller) Install(ctx context.Context, log logr.Logger, cluster *types.Cluster, spec *cluster.Spec, deploymentTimeout string) error {
```

Comment:
should we just make this configurable through the constructor instead?

If you prefer to pass it to the method, can we document what it does in the doc comment. It's not super obvious without reading the code.